src/tab/Tab.tsx

```diff
@@ -19,21 +19,22 @@ interface UncontrolledTabProps {
   testid?: string;
   initialActiveTabIndex?: number;
   customClassName?: string;
+  wrapperId?: string;
 }
 
 // if one of the controlled tab props are present
 // other controlled tab prop is required
 // and initialActiveTabIndex should be undefined
 type ControlledTabProps =
   | {
-      activeTabIndex: number;
-      onTabChange: (index: number) => void;
-      initialActiveTabIndex?: number;
-    }
+    activeTabIndex: number;
+    onTabChange: (index: number) => void;
+    initialActiveTabIndex?: number;
+  }
   | {
-      activeTabIndex?: number;
-      onTabChange?: (index: number) => void;
-    };
+    activeTabIndex?: number;
+    onTabChange?: (index: number) => void;
+  };
```

Comment:
This is totally unrelated to the issue, but I was trying to understand this comment and the type and I couldn't.
Do you (or anyone) know what is our purpose with this type? Why do we need the second part?
```tsx
    activeTabIndex?: number;
    onTabChange?: (index: number) => void;
```

(BTW, code formatting here also looks weird as I mentioned in the other comment)